api/app_sph_lms/api/views.py

```diff
@@ -35,6 +35,7 @@ def get_auth_user(request):
         user = request.user
         token = Token.objects.get(user=user)
         return Response({
+            'full_name': user.first_name + ' ' + user.last_name,
```

Comment:
- you don't need to add `full_name` attribute all the time.
- use https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield